Feature/lp 72 filled text field

NomasChuy opened the pull request:
Hi, this is my current ticket:
https://devaptivist.atlassian.net/browse/LP-72?search_id=c6e022c4-6b19-4ffd-82f6-8e3d3eedb814
I created the DropdownTextField component and added a string for accessibility purposes

Heres an example result of the component:
![image](https://user-images.githubusercontent.com/51238232/231003928-4ae89a5a-6169-45ab-835e-be87984be606.png)

NomasChuy committed:
Merge branch 'feature/48_CardWithImage' into develop

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

NomasChuy committed:
Created the DropdownTextField component and added a string for accessibility purposes

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:
```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
+
+/**
+ * [DropdownTextField] it's a textfield to show a list of items inside a box
+ *
```
Please, add the param text to the documentation and the modifier to the documentation and function.

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:
```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DropdownTextField(
```
I think you need a high order function to get the information of the text field that it is selected

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:
```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
+    ExposedDropdownMenuBox(
+        expanded = expanded,
+        onExpandedChange = { expanded = !expanded },
+        modifier = Modifier.padding(all = 24.dp)
```
please, add the modifier of the params function

NomasChuy committed:
Created the DropdownTextField component and added a string for accessibility purposes

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

NomasChuy committed:
Merge branch 'develop' into feature/lp_72_FilledTextField

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

NomasChuy committed:
Merge branch 'develop' into feature/lp_72_FilledTextField

NomasChuy committed:
 This is a combination of 3 commits.
 This is the 1st commit message:

Created the DropdownTextField component and added a string for accessibility purposes

 The commit message #2 will be skipped:

 Squash 2

 The commit message #3 will be skipped:

 Created the DropdownTextField component and added a string for accessibility purposes

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:
```diff
@@ -167,3 +171,70 @@ fun LpOutlinedTextFieldNumber(
+ * @param title refers to the label that the textfield will have
+ * @param items refers to the list that will be given to expand the textfield
+ * @param modifier to set component modifier
+ * @param
```
You forget to add the param selected, please add the description for this parameter

NomasChuy committed:
Created the DropdownTextField component and added a string for accessibility purposes

carlosph816 reviewed (approved):
Well done!

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

NomasChuy committed:
Merge remote-tracking branch 'origin/develop' into develop

NomasChuy committed:
Merge branch 'develop' into feature/lp_72_FilledTextField

# Conflicts:
#	core/designsystem/src/main/res/values/strings.xml
